src/components/Menu/SocialLinksMenuButton.tsx

```diff
@@ -0,0 +1,77 @@
+"use client"
+
+import * as DropdownMenu from '@radix-ui/react-dropdown-menu'
+import {
+  Github,
+  Instagram,
+  Linkedin,
+  Rocket,
+  Twitter,
+  User,
+  Youtube
+} from 'lucide-react'
+import { MenuButton } from './MenuButton'
+
+export function SocialLinksMenuButton() {
+  const socialLinks = [
+    {
+      name: 'github',
+      url: 'https://github.com/diego3g',
+      icon: <Github size={14} />
+    },
+    {
+      name: 'instagram',
+      url: 'https://instagram.com/dieegosf',
+      icon: <Instagram size={14} />
+    },
+    {
+      name: 'linkedin',
+      url: 'https://www.linkedin.com/in/diego-schell-fernandes/',
+      icon: <Linkedin size={14} />
+    },
+    {
+      name: 'rocketseat',
+      url: 'https://www.rocketseat.com.br/',
+      icon: <Rocket size={14} />
+    },
+    {
+      name: 'twitter',
+      url: 'https://twitter.com/dieegosf',
+      icon: <Twitter size={14} />
+    },
+    {
+      name: 'youtube',
+      url: 'https://www.youtube.com/@rocketseat',
+      icon: <Youtube size={14} />
+    },
+  ]
+
+  return (
+    <DropdownMenu.Root>
+      <DropdownMenu.Trigger>
```

Comment:
Can we add an `aria-label` attribute to this trigger component so screen reader knows what it's for? Maybe something like:
```js
<DropdownMenu.Trigger aria-label="Redes sociais">
```

Reply:
done. Another point that I definitely will work on, at least in my projects.